core/felt/felt.go

```diff
@@ -174,3 +174,33 @@ func (z *Felt) Bits() [4]uint64 {
 func (z *Felt) BigInt(res *big.Int) *big.Int {
 	return z.val.BigInt(res)
 }
+
+// Double forwards the call to underlying field element implementation
+func (z *Felt) Double(x *Felt) *Felt {
+	z.val.Double(&x.val)
+	return z
+}
+
+// Sub forwards the call to underlying field element implementation
```

Comment:
I think we should update all the comments her to depict what the function is doing cos it'll help a lot when reading through codes. I personally had to move through 2 layers of files just to know what a function is doing. It becomes a problem when you have to do this multiple times.

Reply:
ideally, we should get rid of all those wrappers. So it is not really worth to document them properly.